### PR TITLE
 Implement CrossChainSidechainTerminatedCMD cross chain command - Closes #7028 #7030 

### LINKED ISSUE #7028: Implement CrossChainChannelTerminatedCommand
### Description

Implement `CrossChainChannelTerminatedCommand` under `cross_chain_commands` for both `sidechain_interoperability` and `mainchain_interoperability` according to [LIP0049:Channel Terminated Message](https://github.com/LiskHQ/lips/blob/aced79e3a90c893f25c8f0735ce73732f17cced6/proposals/lip-0049.md#channel-terminated-message-1)
- Define cross chain ID as `CROSS_CHAIN_COMMAND_ID_CHANNEL_TERMINATED`
- Define schema as empty
- Implement `exceute(crossChainCommandContext)` according to [LIP0049:Channel Terminated Message](https://github.com/LiskHQ/lips/blob/aced79e3a90c893f25c8f0735ce73732f17cced6/proposals/lip-0049.md#channel-terminated-message-1)
- Make sure `crossChainCommandContext` is defined
- Requires `createTerminatedStateAccount` private function

### Acceptance Criteria

- Should have all the unit tests covering initialisation and execution of the command

### PR DESCRIPTION
**Pull request by Incede**
### What was the problem?

This PR resolves #7028 #7030 

### How was it solved?

Implement commands as specified

### How was it tested?

Added unit tests

### CODE REVIEW

File: framework/src/modules/interoperability/mainchain/cc_commands/channel_terminated.ts [hunk 1]
```diff
@@ -12,17 +12,23 @@
...
 
-import { BaseCCCommand } from '../../base_cc_command';
+import { BaseInteroperabilityCCCommand } from '../../base_interoperability_cc_commands';
 import { CROSS_CHAIN_COMMAND_ID_CHANNEL_TERMINATED } from '../../constants';
 import { channelTerminatedCCMParamsSchema } from '../../schema';
+import { CCCommandExecuteContext, StoreCallback } from '../../types';
+import { MainchainInteroperabilityStore } from '../store';
 
-export class CCChannelTerminatedCommand extends BaseCCCommand {
+export class CCChannelTerminatedCommand extends BaseInteroperabilityCCCommand {
```
Comment (ishantiw) [inline framework/src/modules/interoperability/mainchain/cc_commands/channel_terminated.ts:21]: maybe better to have a prefix `MainchainCCChannelTerminatedCommand` and same for Sidechain and its commands as there will be 2 different classes with the same name

**Review by ishantiw — COMMENTED** [review]
LGTM, just one comment regarding ccCommand naming

File: framework/src/modules/interoperability/mainchain/cc_commands/channel_terminated.ts (hunk 1 above)
Comment (Incede) [reply framework/src/modules/interoperability/mainchain/cc_commands/channel_terminated.ts]: updated

**Review by ishantiw — APPROVED** [review]

**Review by mitsuaki-u — APPROVED** [review]
🚀

**Pull request merged**